index.js

```diff
@@ -24,13 +24,16 @@ WebpackWriter.prototype.build = function () {
 		var compiler = webpack(this.options)
 		return new RSVP.Promise(function (resolve, reject) {
 			compiler.run(function (err, stats) {
-				var jsonStats = stats.toJson()
-				if (jsonStats.errors.length > 0) jsonStats.errors.forEach(console.error)
-				if (jsonStats.warnings.length > 0) jsonStats.warnings.forEach(console.warn)
+        var jsonStats = {warnings: [], errors: []};
```

Comment:
Style does not match the rest of the file, please use tabs for indentation and do not use semicolons